content/chronograf/v1.8/guides/visualization-types.md

```diff
@@ -1,14 +1,15 @@
 ---
 title: Visualization types in Chronograf
-descriptions: Chronograf dashboards and views support graphs and other visualization types, including line graphs, stacked graphs, step-plot graphs, single statistics, bar graphs, gauges, and tables.
+descriptions: Chronograf provides multiple visualization types to visualize your data in
```

Comment:
I prefer to keep line length to around 80 characters to make code reviews easier. When you have a long line, or a paragraph on a single line, in Github, you can only comment on the one line. If you use semantic line endings with a rough line length, comments in PR reviews can be specific to a line/sentence rather than a whole paragraph.

With that said, to do multiline descriptions in YAML, you have to use the multiline string operator (`>`).